fetch.bs

```diff
@@ -1077,6 +1077,9 @@ explicitly set <a for=/>request</a>'s
 <p id=keep-alive-flag>A <a for=/>request</a> has an associated
 <dfn for=request export>keepalive flag</dfn>. Unless stated otherwise it is unset.
 
+<p id=speculative-flag>A <a for=/>request</a> has an associated
+<dfn for=request export>speculative flag</dfn>. Unless stated otherwise it is unset.
```

Comment:
Probably this should be a boolean, as I think we're leaning more towards explicit booleans as opposed to "flags"s these days?